mne/epochs.py

```diff
@@ -508,6 +508,42 @@ def plot(self, epoch_idx=None, picks=None, scalings=None,
                            scalings=scalings, title_str=title_str,
                            show=show, block=block)
 
+    def plot_concat(self, picks=None, scalings=None, n_epochs=8,
+                    n_channels=10, title_str='#%003i', show=True, block=False):
```

Comment:
yeah, but the deprecations must be handled carefully. The old function could be called `plot_trellis` or something if we still want to keep it. @jaeilepp are you up for that?